Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs

```diff
@@ -121,10 +121,12 @@ public override string GetDriverConfigsEditPageName(Agent.eDriverType driverSubT
         [UserConfiguredDefault("true")]
         [UserConfiguredDescription("Auto Detect Proxy Setting?")]
         public bool AutoDetect { get; set; }
+
         [UserConfigured]
         [UserConfiguredDefault("")]
-        [UserConfiguredDescription("Path to extension to be enabled")]
-        public string ExtensionPath { get; set; }
+        [UserConfiguredDescription("Path of extensions to be enabled | Use a semi-colon to separate multiple extensions.")]
+        public string ExtensionPaths { get; set; }
```

Comment:
Please verify if it will still use the previous values used in old solution with 'ExtensionPath'

Reply:
@Maheshkale447  Then maybe we can just keep the previous `ExtensionPath` property name? Because the configuration options are inside the Agent files themselves, in order see this new property or any updates, Ginger users would have to create a new Agent file. 

Maybe, If we reuse the name we could just __extend__ it behind the scenes and allow multiple with `;`.

Reply:
Please check Serialization error method in Agent.cs class. if there is missing property, this method should catch that error. there you can copy value from old property to new property.

Reply:
@Maheshkale447 Checked but don't see the `SerializationError` error being thrown using the latest Chrome drivers (v119+) and using the an older Agent config file. Maybe, the serialization is implemented differently as Ginger recently switched selenium versions?

Reply:
Let's keep it as ExtensionPath instead of ExtensionPaths as of now. This page will be redesigned in coming releases, name change will be handled as part of the redesign

Reply:
Sure @Maheshkale447 , changed it back.